core/src/system_monitor_service.rs

```diff
@@ -832,6 +836,47 @@ impl SystemMonitorService {
         Self::report_cpuid_values();
     }
 
+    fn get_open_fd_stats() -> Option<(usize, usize, usize, usize)> {
+        let proc = Process::myself().ok()?;
+        let curr_num_open_fd = proc.fd_count().unwrap();
+        let curr_mmap_count = proc.maps().unwrap().len();
```

Comment:
this will measure _all_ of the processes maps, not just appendvecs, so will be slightly inflated

Reply:
yeah. 
jeff has another change to track just appendvecs.
https://github.com/solana-labs/solana/pull/28958